environment.yml

```diff
@@ -1,23 +0,0 @@
-name: ramp
-dependencies:
-  - numpy
-  - pyyaml
```

Comment:
Why do you delete this one?

Reply:
Ah, or you only renamed it to `test_environment.yml` ? (for me, just `environment.yml` is fine for the base environment to set up the full server/frontend/database/worker thing)

Reply:
I renamed it test_environment.yml. I can revert the renaming

Reply:
BTW, the `test_environment.yml` file seems empty

Reply:
Uhm. What do you mean?

Reply:
it just looked empty in the diff, but now you renamed it again, the diff looks good